client/src/components/filters/text/TextSearchField.jsx

```diff
@@ -16,6 +16,15 @@ const styleSearchFieldFocused = {
   boxShadow: 'inset 0 2px 4px rgba(0, 0, 0, 0.5)',
 }
 
+const styleTextFieldLabel = {
```

Comment:
could this be simplified to:

```
const styleTextFieldLabel = {
  width: '0',
  height: '0',
  overflow: 'hidden',
}
```

You might need to add `display: 'inline'` to prevent a break, in that case, but I would make sure  if that break occurs with 0x0 pixel elements. Not 100% on the behavior here in general and across browsers.

Reply:
You can't size it to 0 or the screen reader misses it. Good catch and suggestion though. It is what I initially wanted but didn't work. The css I ended up using was straight from webaim- 
https://webaim.org/techniques/css/invisiblecontent/#techniques